instance/models/openedx_instance.py

```diff
@@ -146,6 +147,21 @@ def studio_domain(self):
             return self.internal_studio_domain
 
     @property
+    def studio_domain_nginx_regex(self):
+        """
+        Regex that matches either the internal or the external Studio URL.
+
+        This is exclusively meant for the Ansible variable CMS_HOSTNAME to configure the nginx
+        server_name regex to match the Studio domains.
+        """
+        if self.external_studio_domain:
+            domains = [self.external_studio_domain, self.internal_studio_domain]
+        else:
+            domains = [self.internal_studio_domain]
+        choices = '|'.join(map(re.escape, domains))  # pylint: disable=bad-builtin
```

Comment:
@smarnach I personally can see their argument, and find the list comprehension clearer. Ultimately, most of these cases will be a matter of personal preference, there will often be arguments that can go in both cases. Taking the approach of just following pylint recommendations has the advantage of providing a ready-made coding style, while nullifying the need for debates on these, which in turn allows us to spend time on more important things.